presto-plugin-toolkit/src/test/resources/table.json

```diff
@@ -13,9 +13,11 @@
       "user": "bob",
       "schema": "bobschema",
       "table": "bobtable",
+      "column": "(bobcolumn|bobcredential)",
       "privileges": ["SELECT", "INSERT", "DELETE", "GRANT_SELECT"]
     },
     {
+      "column": "bobcolumn",
```

Comment:
I would divide this commit into two. First I would introduce ability to set privileges for columns and implemented database object filtering.

Or even better I would introduce  privileges for columns as separate PR, IMO it requires some discussion and looking into SQL specification because I am not sure it is going to work with `io.prestosql.plugin.base.security.TableAccessControlRule.TablePrivilege`. For example can user DELETE data from table when they have access only to some columns?

So for now you could only implement filtering for schemas and tables.